### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by mkvillalobos** [alanross/AlvaAR#25]
Hello!

I hope you be fine!

This project is REALLY amazing, but seems it doesn't have any progress since a lot of months ago. 

I would like to know if this project is still on development or, well, already abandoned.  I really hope not!!!

Thank you so much!!

Best regards!

**Comment by alanross** [comment]
Hello @mkvillalobos, I appreciate your kind words. The project is ongoing; however, my day job has kept me busy and from revisiting key areas like merging visual and IMU data to get more robust positioning. I plan to get back to this work as soon as I find some time.

**Comment by mkvillalobos** [comment]
Hello @alanross !  Thank you so much for your reply!

I can understand you perfectly!  And knowing that the project is ongoing are great news: I really think is truly unique... bring SLAM to the browser! 

Thanks for all!  Sucess in your projects and your job!

Best regards!

**Comment by marcusx2** [comment]
@alanross when you get back to it please don't forget [these requests](https://github.com/alanross/AlvaAR/issues/12#issuecomment-1676003926).

**Comment by DouglasLapsley** [comment]
Yeah, this is a holy grail for AR.  Really cracking this nut would be greatly appreciated.

**Comment by nguyenph194812** [comment]
@alanross hello and wish u have a good day. i waana ask that when will you be back on this project. Without IMU data the object sometimes jummping all over the scene. And may i was wondering: Why ORB-slam2 but not ORB-slam3 ?

**Comment by YulesRules** [comment]
Wishing you well. This is currently the forefront of open source web XR. We're all rooting for you.

**Comment by coopbri** [comment]
I'm curious if funding would change things? We're looking to integrate web-based SLAM and VPS into https://github.com/omnidotdev/rdk and, well, it's a complicated task. Have you thought about taking donations or etc.?
